test/addTwo.spec.ts

```diff
@@ -0,0 +1,7 @@
+import * as path from "path";
+import {getWasmInstance} from "./utils/utils";
+
+test('it should add two numbers', () => {
+    const instance:WebAssembly.Instance = getWasmInstance(path.join(__dirname, 'addTwo.tbs'), path.join(__dirname, 'bin/addTwo.wasm'));
```

Comment:
I'd suggest the destination file be in a `tmp` directory and then add `tmp` directory to `.gitignore`